Assets/Scripts/HostOrJoinScene/TabSelection.cs

```diff
@@ -0,0 +1,32 @@
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+
+public class TabSelection : MonoBehaviour
+{
+    private EventSystem system;
+
+    private void Start() {
+        system = EventSystem.current;
+    }
+
+    private void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.Tab))
+        {
+            Selectable target;
+
+            target = system.currentSelectedGameObject?.GetComponent<Selectable>().FindSelectableOnDown();
```

Comment:
These lines can be merged.

```suggestion
            var target = system.currentSelectedGameObject?.GetComponent<Selectable>().FindSelectableOnDown();
```